doc/source/index.rst

```diff
@@ -40,17 +40,202 @@ An example of such a callable class used in trollflow2 is the
 `class:FilePublisher`.
 
 
-Available plugins
-+++++++++++++++++
+Main plugins
+++++++++++++
 
 .. currentmodule:: trollflow2.plugins
 
-The `check_sunlight_coverage` plugin
-************************************
+Scene creation
+**************
+
+The `create_scene` plugin creates a scene object.  The configuration
+options are:
+
+ - `reader` - Name of the reader to use to open the data.  If not given, Satpy will try to find the reader automatically.
+ - `reader_kwargs` - A dictionary of special keyword arguments passed
+   to the reader.
+ - `ppp_config_dir` - Path to the Pytroll configuration directory.
+   If not given, environment variable `$PPP_CONFIG_DIR` is used.
+
+This plugin needs to be defined before data are accessed in any way.
+
+Dataset/composite loading
+*************************
+
+After the scene has been created with `create_scene`, composites and
+other datasets can be loaded with `load_composites`. The
+composites/datasets are defined in the product list YAML file under
+`products` section within a section of target area.  There are two
+special keyword arguments:
+
+ - `resolution` - If there are several resolutions of the same
+   dataset, this argument can be defined to select the correct one.
+   By default the highest resolution data are used.
+ - `delay_composites` - The composite generation can be delayed until
+   resampling.  This can save a lot of time if the same channels are
+   used in several composites.  Default: `True`.
+
+Resampling
+**********
+
+After the data are loaded, it can be resampled with `resample` plugin.
+There are multiple different options depending on the resampler.  For
+complete list of resampler options see documentation of `Satpy
+resampling <https://satpy.readthedocs.io/en/latest/resample.html>`_.
+
+The default values are:
+
+ - `resampler: nearest`
+ - `radius_of_influence: null`
+ - `reduce_data: True`
+ - `cache_dir: None`
+ - `mask_area: False`
+ - `epsilon: 0.0`
+
+For dynamic area definitions, which are not named, one can define the area as `null` and define one of
+
+ - `use_min_area: True` or
+ - `use_max_area: True`
```

Comment:
I wouldn't call this a dynamic area definition, it's just the satellite projection with pixels aggregated or repeated to make the mulitple resolutions match.

Reply:
We discussed this misunderstanding in Slack, and what the `min_area` and `max_area` really mean, and the confusion the adopted naming for them cause (`area` when they really are a matter of `resolution`). I'll rephrase and point to Satpy/Pyresample docs.